packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,844 @@
+            special use case.</span>
+        </div>
+      </:actions>
+    </Hds::Alert>
+  </div>
+
+  <h4 class="dummy-h4">Dismiss</h4>
+  <div class="dummy-alert-sample-grid dummy-alert-sample-grid--wide-content">
+    <Hds::Alert
+      @type="inline"
+      @color="critical"
+      @title="Without the dismiss button (default)"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+    />
+    <Hds::Alert
+      @type="inline"
+      @color="critical"
+      @title="With the dismiss button"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+      {{! TODO: understand if we can use a generic helper - see https://hashicorp.slack.com/archives/C11JCBJTW/p1648751235987409 }}
+      @onClose={{this.noop}}
+    />
+    <Hds::Alert
+      @type="inline"
+      @color="critical"
+      @icon=""
+      @title="With the dismiss button and no icon"
+      @description="Lorem ipsum dolor sit amet, consectetur adipiscing elit."
+      @onClose={{this.noop}}
+    />
+    <Hds::Alert
+      @type="inline"
+      @color="critical"
+      @description="With the dismiss button and no title"
+      @onClose={{this.noop}}
+    />
+  </div>
+
+</section>
+
+<section>
+  <h3 class="dummy-h3">Accessibility</h3>
+  <p class="dummy-paragraph">Dynamically-rendered Alerts are automatically announced by most screen readers. On some
+    operating systems, Alerts may trigger an alert sound.
+  </p>
+
+  <p class="dummy-paragraph">🚧 TODO: Have Melanie review post-Amy transfer</p>
+  <p class="dummy-paragraph">🚧 TODO: Bring role change to code, if folks agree</p>
+  <ul>
+    <li class="dummy-paragraph">
+      If
+      <code class="dummy-code">color="warning"</code>
+      or
+      <code class="dummy-code">color="critical"</code>
+      => then
+      <code class="dummy-code">role="alert"</code>, as it's an important message.
+    </li>
+    <ul>
+      <li class="dummy-paragraph">Reference:
+        <a href="https://www.w3.org/TR/wai-aria-practices-1.2/#alert" target="_blank" rel="noopener noreferrer">
+          WAI-ARIA Authoring Practices 1.2 (Alert)
+        </a>
+      </li>
+    </ul>
+    <li class="dummy-paragraph">
+      If
+      <code class="dummy-code">color="neutral"</code>,
+      <code class="dummy-code">color="highlight"</code>
+      or
+      <code class="dummy-code">color="success"</code>
+      => then
+      <code class="dummy-code">role="status"</code>, as the content is advisory information for the user, but is not
+      important enough to justify a
+      <code class="dummy-code">role="alert"</code>. Elements with
+      <code class="dummy-code">role="status"</code>
+      have an implicit
+      <code class="dummy-code">aria-live="polite"</code>
+      and
+      <code class="dummy-code">aria-atomic="true"</code>, so we have included them explicitly, for clarity.
+    </li>
+    <ul>
```

Comment:
This must also be inside of an `<li>` element (see explanation above)

Reply:
removed content, not relevant anymore